src/OpenTelemetry.AutoInstrumentation/InstrumentationDefinitions.cs

```diff
@@ -0,0 +1,90 @@
+// <copyright file="InstrumentationDefinitions.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+namespace OpenTelemetry.AutoInstrumentation;
+
+internal static partial class InstrumentationDefinitions
+{
+    private static string assemblyFullName = typeof(InstrumentationDefinitions).Assembly.FullName!;
+
+    internal static Payload GetAllDefinitions()
+    {
+        return new Payload
+        {
+            // Fixed Id for definitions payload (to avoid loading same integrations from multiple AppDomains)
+            DefinitionsId = "FFAFA5168C4F4718B40CA8788875C2DA",
+
+            // Autogenerated definitions array
+            Definitions = GetDefinitionsArray(),
```

Comment:
It is not directly related to this line, but I think it best fits here.

After changes instrumentation are injected from managed-code to native-code.
What is more on the managed code we check which instrumentations are enabled.
Similar code is written in native code.
It will be great to to pass to the native code only instrumentation really enabled and fully remove code related to enabled/disabled instrumentation from the native code part.

If you agree, we can convert it to separate GitHub issue.